daisy/logger.go

```diff
@@ -159,6 +170,14 @@ func (l *daisyLog) WriteSerialPortLogs(w *Workflow, instance string, buf bytes.B
 	writeLog(str)
 }
 
+func (l *daisyLog) ReadSerialPortLogs() []string {
+	logs := make([]string, 0, len(l.serialLogs))
```

Comment:
We lose the information of which instance has logged a particular serial log (e.g. disk importer vs translator). In most cases we might be able to figure which one is which, but logging instance name in some way would reduce the workload on our side and uniquely identify which instance has logged a particular log file. It could be as simple as adding instance name with a specific prefix as the first line of each serial log.

Reply:
Good idea! Added that.